### ISSUE TITLE
Images went black

### ISSUE THREAD

**Issue by Jezz3** [ptrLx/OneShot#17]
I think after the recent update some of my images went dark.

They are still jn the Oneshot folder but show blacl in the app.

![Screenshot_20230409_094617_OneShot](https://user-images.githubusercontent.com/37677169/230760889-2d54af83-535d-4ada-be7d-82a44ce6806f.jpg)

**Comment by ptrLx** [comment]
That's strange... Are the images also black in the large image view?
(Click on the image and then again on it in the modal bottom sheet)

**Comment by Jezz3** [comment]
yes, i also tried to export a backup, clear the data and cache of the app and load the backup again
